src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -40,24 +44,29 @@ const ForgotPasswordPage = (props) => {
 
   return (
     <Formik
+      validate={(values) => {
+        if (values.email === '') {
+          return { email: intl.formatMessage(messages['forgot.password.empty.email.field.error']) };
+        }
+        return {};
+      }}
       onSubmit={(values) => {
         if (values.isEmailValid) {
           props.forgotPassword(values.email);
         }
+        mainRef.current.focus();
       }}
       initialValues={{
         email: '',
         isEmailValid: true,
       }}
     >
       {({
-        handleSubmit,
-        values,
-        setFieldValue,
+        errors, handleSubmit, setFieldValue, values,
       }) => (
         <>
           {status === 'complete' ? <Redirect to={LOGIN_PAGE} /> : null}
-          <div className="d-flex justify-content-center m-4">
+          <div className="d-flex justify-content-center m-4" tabIndex="-1" ref={mainRef}>
```

Comment:
`tabIndex` is adding a strange blue border in Safari, can we remove this?

Reply:
One of the requirements of the ticket is:
> On submit a response; if errors are generated take page focus to the top of the page on errors alert box.

I added tab index to make it focusable using ref.